app/locales/fr/translations.js

```diff
@@ -320,6 +320,10 @@ export default {
     'badCredentials': "Nom d'utilisateur ou mot de passe incorrect",
     'noAccountExists': "Votre compte d'utilisateur {{accountName}} ne correspond pas à ceux des utilisateur enregistre en Ilios. Si vous avez besoin d'aide, n'hesitez pas à communiquer avec votre administrateur  d'Ilios.",
   },
+  'user': {
+    'firstName': 'Prénom',
+    'lastName': 'Nom de Famille',
+  },
```

Comment:
Google is A-OK, but use sentence case not title case: "Nom de famille", and "Prénom".